<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-AP8LOSU" failures="1" tests="1" name="Default test(failed)(failed)" time="0.13" errors="0" timestamp="2024-12-17T23:01:12 IST">
  <testcase classname="test.TestNgFailedDemo" name="test2" time="0.022">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
at test.TestNgFailedDemo.test2(TestNgFailedDemo.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase classname="test.TestNgFailedDemo" name="test1" time="0.0">
    <ignored/>
  </testcase> <!-- test1 -->
  <testcase classname="test.TestNgFailedDemo" name="test3" time="0.0">
    <ignored/>
  </testcase> <!-- test3 -->
</testsuite> <!-- Default test(failed)(failed) -->
